ghost/i18n/locales/uk/portal.json

```diff
@@ -0,0 +1,60 @@
+{
+    "{{discount}}% discount": "Знижка {{discount}}%",
+    "{{trialDays}} days free": "{{trialDays}} днів безкоштовно",
+    "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "Посилання для входу було надіслано на твою пошту. Якщо воно не прийде протягом 3 хвилин, перевірь папку спам.",
+    "Account": "Oбліковий запис",
+    "Account settings": "Налаштування облікового запису",
+    "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "Після закінчення безкоштовного тестування, буде зараховано звичайну ціну за вибраний план. Завжди можеш скасувати перед тим як це станеться.",
+    "Already a member?": "Вже зареєстрований?",
+    "Back": "Назад",
+    "Back to Log in": "Повернутись до входу",
+    "Cancel subscription": "Скасуй підписку",
+    "Cancellation reason": "Причина скасування",
+    "Choose a different plan": "Вибари інший план",
+    "Choose your newsletters": "Вибери свої підписки",
+    "Close": "Закрий",
+    "Comments": "Коментарі",
+    "Confirm": "Підтвердь",
+    "Continue": "Далі",
+    "Delete account": "Усунь обліковий запис",
+    "Don't have an account?": "Не маєш облікового запису?",
+    "Email": "Електронна пошта",
+    "Email preference updated.": "Налаштування електронної пошти оновлені.",
+    "Email preferences": "Налаштування електронної пошти",
+    "Emails": "Електронні листи",
+    "Emails disabled": "Електронна пошта вимкнена",
+    "Get help": "Отримати допомогу",
+    "Get notified when someone replies to your comment": "Повідомляй коли хтось відповість на твій коментар",
+    "Give feedback on this post": "Дати відгук на цей пост",
+    "Less like this": "Менше подібних",
+    "Manage": "Управління",
+    "Monthly": "Місячно",
+    "More like this": "Більше подібних",
+    "Name": "Ім'я",
+    "Not receiving emails?": "Не приходять листи?",
+    "Now check your email!": "А тепер перевір свою пошту!",
+    "Price": "Ціна",
+    "Re-enable emails": "Знову включити пошту",
+    "Retry": "Повтори спробу",
+    "Save": "Збережи",
+    "Sending login link...": "Відправляється посилання для входу...",
+    "Sending...": "Відправляється...",
+    "Sign in": "Вхід",
+    "Sign up": "Вихід",
+    "Start {{amount}}-day free trial": "Почни {{amount}}-денний термін безкоштовного тестування",
```

Comment:
I suggest to use `Почни {{amount}}-денний безплатний період`, since it more accurately expresses the intended concept. In this context, _"безкоштовний"_ implies something without cost or value, whereas _"безплатний"_ means it is not requiring payment or reward. Using _"безплатний"_ more accurately expresses the concept of a free trial period.